flyteplugins/go/tasks/plugins/k8s/batchscheduler/plugins.go

```diff
@@ -0,0 +1,23 @@
+package batchscheduler
+
+import (
+	"context"
+	"github.com/flyteorg/flyte/flyteplugins/go/tasks/plugins/k8s/batchscheduler/scheduler"
+	"github.com/flyteorg/flyte/flyteplugins/go/tasks/plugins/k8s/batchscheduler/scheduler/yunikorn"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type SchedulerManager interface {
```

Comment:
How about docs that say,something to this effect)
```suggestion
// SchedulerManager is an interface that allows plugging in custom schedulers 
// such as Yunikorn, Kueue, or any other scheduler that manages pod scheduling 
// with advanced features like gang scheduling, preemption, and priority.
type SchedulerManager interface {
```